packages/atomic/src/components/recommendations/atomic-recs-result/atomic-recs-result.tsx

```diff
@@ -0,0 +1,179 @@
+import {Component, h, Prop, Element, Listen} from '@stencil/core';
+import {applyFocusVisiblePolyfill} from '../../../utils/initialization-utils';
+import {
+  DisplayConfig,
+  ResultContextEvent,
+} from '../../search/result-template-components/result-template-decorators';
+import {
+  ResultLayout,
+  ResultDisplayDensity,
+  ResultDisplayImageSize,
+  ResultDisplayLayout,
+} from '../../common/layout/display-options';
+import {RecsResult} from '..';
+import {AtomicRecsStore} from '../atomic-recs-interface/store';
+import {ResultRenderingFunction} from '../../common/result-list/result-list-common-interface';
+
+/**
+ * The `atomic-recs-result` component is used internally by the `atomic-recs-list` component.
```

Comment:
Should the component itself be marked as internal, so it doesn't generate public docs?

Reply:
Oh yes!